### ISSUE TITLE
About pad_labels function

### ISSUE THREAD

**Issue by wincing2** [k2-fsa/ZipVoice#52]
I'm studying the code and I'm confused by the first line of pad_labels. Why does it append pad_id to each sequence, including the longest sequence, instead of just padding shorter sequences to the longest sequence? 

```
def pad_labels(y: List[List[int]], pad_id: int, device: torch.device):
    """
    Pad the transcripts to the same length with zeros.

    Args:
      y: the transcripts, which is a list of a list

    Returns:
      Return a Tensor of padded transcripts.
    """
    y = [token_ids + [pad_id] for token_ids in y]
    length = max([len(token_ids) for token_ids in y])
    y = [token_ids + [pad_id] * (length - len(token_ids)) for token_ids in y]
    return torch.tensor(y, dtype=torch.int64, device=device)
```
